iwyu.cc

```diff
@@ -4362,12 +4367,17 @@ class IwyuAstConsumer
 
 // We use an ASTFrontendAction to hook up IWYU with Clang.
 class IwyuAction : public ASTFrontendAction {
+ public:
+  IwyuAction(const ToolChain& toolchain) : toolchain(toolchain) {
+  }
+
```

Comment:
Explicitly delete the default constructor?

Reply:
Good idea! Also made the toolchain ctor explicit.